reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -664,44 +605,61 @@ pwsh
 
 ```sh
 # Grab the latest App Image
-wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0/powershell-6.0.0-x86_64.AppImage
+wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/powershell-6.0.2-x86_64.AppImage
 
 # Make executable
-chmod a+x powershell-6.0.0-x86_64.AppImage
+chmod a+x powershell-6.0.2-x86_64.AppImage
 
 # Start PowerShell
-./powershell-6.0.0-x86_64.AppImage
+./powershell-6.0.2-x86_64.AppImage
 ```
 
 ### Uninstallation - Kali
 
 ```sh
-sudo dpkg -r powershell-6.0.0-x86_64.AppImage
+dpkg -r powershell_6.0.2-1.ubuntu.16.04_amd64.deb
```

Comment:
sudo not required?

Reply:
I had the same question but sudo was not in the example in https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md#kali